ddtrace/profiling/collector/stack.pyx

```diff
@@ -344,8 +345,8 @@ cdef stack_collect(ignore_profiler, thread_time, max_nframes, interval, wall_tim
                 thread_id=thread_id,
                 thread_native_id=thread_native_id,
                 thread_name=thread_name,
-                task_id=thread_task_id,
-                task_name=thread_task_name,
+                task_id=None,
+                task_name=None,
```

Comment:
In the logical abstractions of tasks on top of threads, not all threads will have a task running within them. Besides, since with the module cloning approach we would have the ddtrace threads running as Python threads, we can no longer assign the current task data to all threads, else we would end up reporting e.g. that the stack collector thread is running a gevent task.

Reply:
a code comment would be a nice to have since this isn't obvious (at least to me) from the code alone.

Reply:
Wall time accounting is part of an ongoing discussion. I think we'll have chances to get back to this with more details. I will postpone this task until we have more defined details if that's ok.